utils/db/db_connections

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    production: {
+        client: "pg",
+        connection: process.env.DATABASE_URL,
+        pool: {
+            afterCreate: function(connection, callback) {
+                connection.query("SET TIME ZONE 'UTC';", function(err) {
+                    callback(err, connection);
+                });
+            }
+        }
+    },
+	development: {
+				client: "pg",
+				connection: process.env.DATABASE_URL,
+				pool: {
+					afterCreate: function(connection, callback) {
+						connection.query("SET TIME ZONE 'UTC';", function(err) {
+							callback(err, connection);
+						});
+					}
+				}
+			},
+    test: {
+        client: "pg",
+				connection: process.env.DATABASE_TEST_URL,
+				pool: {
+					afterCreate: function(connection, callback) {
+						connection.query("SET TIME ZONE 'UTC';", function(err) {
+							callback(err, connection);
+						});
+					}
+				}
+			}
+}
```

Comment:
Did you want to clean up the spacing before merging?

Reply:
Yeah, there's also some tabs in there and the file is named incorrectly. I'll fix it up.

Reply:
Should be better now. Thanks @mfrederickson.

Reply:
Thank you... I'm learning a lot by reading through the PRs and looking up stuff.